tests/unit/services/date-test.js

```diff
@@ -9,13 +9,25 @@ module('Unit | Service | date', function(hooks) {
   });
 
   test('it returns a date from the now method', function(assert) {
+    assert.expect(1);
```

Comment:
I don't tend to use `assert.expect`. Can we reverse this?

Reply:
:-(

It's great practice, but since theres no asyc here OK